ansible/roles/dev/templates/run_tests.sh.j2

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cd {{ root_dir }}/build
+export PYTHONPATH='{{ root_dir }}/miniconda/lib/python2.7/site-packages:{{ root_dir }}/miniconda/lib/python2.7/site-packages/ctk_cli-1.3-py2.7.egg:$PYTHONPATH'
+cmake ../girder
+make -j2
+ctest -j3 -V
```

Comment:
Since this is a convenience script, I'll recommend letting command line parameters through to the ctest command.  That is, the last line would be changed to:
```
if [ $# -eq 0 ]; then
  ctest -j3 -V
else
  ctest "$@"
fi
```
So that it is easy to run a single tests as `ctest -VV -R mytest_regex`.